warehouse/slave.go

```diff
@@ -485,6 +485,31 @@ func processStagingFile(job Payload, workerIndex int) (loadFileUploadOutputs []l
 			columnType := columnInfo.Type
 			columnVal := columnInfo.Value
 
+			if job.DestinationType == warehouseutils.CLICKHOUSE && model.SchemaType(columnType) == model.BooleanDataType {
+				newColumnVal := 0
+				if k, ok := columnVal.(bool); ok {
+					if k {
+						newColumnVal = 1
+					}
+				}
```

Comment:
Should we add case where the ok is false ? Or would there ever be case like that ?

Reply:
We are already capturing this in the testdata we have..